docs/extensions_dev.md

```diff
@@ -162,7 +162,7 @@ A theme is a JupyterLab extension that uses a `ThemeManager` and can be
 loaded and unloaded dynamically.  The package must include all static assets 
 that are referenced by `url()` in its CSS files.  The `url()` paths in a CSS
 file served by the Jupyter server must start with the path 
-`'./lab/api/themes/<foo>/', where `foo` is the normalized name of the 
+`'./lab/api/themes/<foo>/`, where `foo` is the normalized name of the 
```

Comment:
I think we need both the quote character and the accent character

Reply:
Whoops! Good catch.